envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,20 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    // [#not-implemented-hide:]
+    // Configures percentage of requests that will be routed to same zone
+    // If not specified, the default is 100%.
+    // :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`.
+    // :ref:`Zone aware routing support <arch_overview_load_balancing_zone_aware_routing>`.
+    core.Percent zone_aware_routing_enabled = 2;
+    // [#not-implemented-hide:]
+    // Configures minimum upstream cluster size required for zone aware routing
+    // If upstream cluster size is less than specified ,zone aware routing is not performed
```

Comment:
Nit: s/d ,zone/d, zone/

Reply:
Hello @htuch  . Sorry did not get that comment . can you elaborate? Thanks!

Reply:
Thats @htuch 's talk to sed command :).. He is saying move the comma at the end of the word and not the beginning of another word (aka "specified, zone" instead of "specified ,zone").

Reply:
Got it @rshriram